### DISCUSSION TITLE
Conda installation

### DISCUSSION THREAD

**Discussion by jaredjeya** [QuSpin/QuSpin#713] [Q&A]
With the new 1.0.0 (congrats!), is Conda installation still supported? It's generally much easier to resolve dependencies etc. with conda than pip so I'd prefer to be able to install it that way!

**Comment by mgbukov** [comment]
conda installation is outdated starting with `quspin 1.0.0` (see documentation)

However, any conda environment comes with `pip` so u may install `quspin` using pip within a conda env.

**Comment by jaredjeya** [comment]
Are there any plans to support Conda installation in the future, please? There are a number of known issues with mixing pip and conda installs (https://www.anaconda.com/blog/using-pip-in-a-conda-environment).

The dependencies are also no longer documented which will make doing this kind of installation tricky. If I just use pip to resolve them, then that potentially poses problems if later I also want to install some other package in the same environment (e.g. SLEPc), as the dependencies won't necessarily be recognised by conda. It's strongly suggested in the conda documentation that I install dependencies using Conda instead. Could you please list the dependencies and their versions in the installation instructions?

**Comment by mgbukov** [comment]
I think @weinbe58 wanted to look into a conda installation at some point (but likely not right now, so if u want to take a look into this, go ahead)

The dependencies are listed in the [`pyptoject.toml` file](https://github.com/QuSpin/QuSpin/blob/89bb61f6b05bc092f83fc3793917c6c47471deb0/pyproject.toml)
